src/app/shared/header/header.component.html

```diff
@@ -56,7 +56,10 @@
       <div *ngIf="accountService.isUserLoggedIn()">
         <button mat-menu-item>Account</button>
         <button mat-menu-item>Orders</button>
-        <button mat-menu-item>Sell</button>
+        <button
+          mat-menu-item
+          routerLink="/sell"
```

Comment:
you only added the route here, which is for mobile. I can't access the sell component page unless i switch to mobile. You need to add this route to the desktop view as well which is above this code.